extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheJpaEntityEnhancer.java

```diff
@@ -121,198 +134,38 @@ public MethodVisitor visitMethod(int access, String methodName, String descripto
-                    Opcodes.ARETURN, null, "query", "sort", "params");
-
-            // findAll Sort?
-
-            generateMethod("findAll",
-                    "()" + QUERY_SIGNATURE,
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">()L" + QUERY_BINARY_NAME + "<TT;>;",
-                    Opcodes.ARETURN, null);
-
-            generateMethod("findAll",
-                    "(" + SORT_SIGNATURE + ")" + QUERY_SIGNATURE,
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">(" + SORT_SIGNATURE + ")L" + QUERY_BINARY_NAME + "<TT;>;",
-                    Opcodes.ARETURN, null, "sort");
-
-            // listAll Sort?
-
-            generateMethod("listAll",
-                    "()Ljava/util/List;",
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">()Ljava/util/List<TT;>;",
-                    Opcodes.ARETURN, null);
-
-            generateMethod("listAll",
-                    "(" + SORT_SIGNATURE + ")Ljava/util/List;",
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">(" + SORT_SIGNATURE + ")Ljava/util/List<TT;>;",
-                    Opcodes.ARETURN, null, "sort");
-
-            // streamAll Sort?
-
-            generateMethod("streamAll",
-                    "()Ljava/util/stream/Stream;",
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">()Ljava/util/stream/Stream<TT;>;",
-                    Opcodes.ARETURN, null);
-
-            generateMethod("streamAll",
-                    "(" + SORT_SIGNATURE + ")Ljava/util/stream/Stream;",
-                    "<T:" + ENTITY_BASE_SIGNATURE + ">(" + SORT_SIGNATURE + ")Ljava/util/stream/Stream<TT;>;",
-                    Opcodes.ARETURN, null, "sort");
-
-            // count [String, Map|Object[]|Parameters?]?
-
-            generateMethod("count", "(Ljava/lang/String;[Ljava/lang/Object;)J", null, Opcodes.LRETURN, null, "query", "params");
-            generateMethod("count", "(Ljava/lang/String;Ljava/util/Map;)J", null, Opcodes.LRETURN, null, "query", "params");
-            generateMethod("count", "(Ljava/lang/String;" + PARAMETERS_SIGNATURE + ")J", null, Opcodes.LRETURN, null, "query",
-                    "params");
-            generateMethod("count", "()J", null, Opcodes.LRETURN, null);
-
-            // delete [String, Map|Object[]|Parameters?]?
-
-            generateMethod("delete", "(Ljava/lang/String;[Ljava/lang/Object;)J", null, Opcodes.LRETURN, null, "query",
-                    "params");
-            generateMethod("delete", "(Ljava/lang/String;Ljava/util/Map;)J", null, Opcodes.LRETURN, null, "query", "params");
-            generateMethod("delete", "(Ljava/lang/String;" + PARAMETERS_SIGNATURE + ")J", null, Opcodes.LRETURN, null, "query",
-                    "params");
-            generateMethod("deleteAll", "()J", null, Opcodes.LRETURN, null);
+            for (MethodInfo method : panacheEntityBaseClassInfo.methods()) {
+                AnnotationInstance bridge = method.annotation(JandexUtil.DOTNAME_GENERATE_BRIDGE);
+                if (bridge != null)
```

Comment:
do we require braces?  (please say yes)

Reply:
I'll say not if the formatter does not, and it does not ;)